tests/testthat/test-test_r_scripts.R

```diff
@@ -4,3 +4,14 @@ test_that("stop_on_error stops on error", {
   expect_error(stop_on_error(exit_code = 1), "error")
   expect_error(stop_on_error(exit_code = 99), "error")
 })
+
+test_that("example data is available under working_dir/", {
+  # https://git.io/JkX5O
+  config <-
+    path_from(parameter_file(), "TestPortfolio_Input_PortfolioParameters.yml")
+  portfolio_name <- get_param("parameters", "portfolio_name_in")(config)
+
+  example_data <- path_from(raw_inputs(), glue::glue("{portfolio_name}.csv"))
+  expect_true(fs::file_exists(example_data))
+})
```

Comment:
Here I sacrificed readability in order to try avoid hard-coding paths. I'm not 100% sure this is the right decision and criticism is welcome.

Reply:
My intent here is to test something that the web-tool needs without actually touching the web-tool script. This would be probably be best replaced (or complemented) with an assertion in the web-tool script itself (and I'll follow up with a PR) but this seems the safest move until we have more tests covering this code-base.

Reply:
I'm also considering testing not that the file exists at a given path but instead that the data is available. That way the test should better resist a refactoring by which we become independent of the file system and instead use a more portable way to interface between the different processes.